src/browser-polyfill.js

```diff
@@ -10,7 +10,7 @@ if (typeof window != "object" || typeof chrome != "object" || !chrome || !chrome
   throw new Error("This script should only be loaded in a browser extension.");
 }
 
-if (typeof browser === "undefined" || Object.getPrototypeOf(browser) !== Object.prototype) {
+if (typeof window.browser === "undefined" || Object.getPrototypeOf(window.browser) !== Object.prototype) {
```

Comment:
This part seems to be failing the test and I'm not sure why that is. How can it pass with `browser` but not `window.browser`?

Reply:
In Firefox's content scripts, the global is not the same as window. Conceptually you can view the global as an object (global) with its prototype set to `window`.

Reply:
Interesting. What could be a solution here? `globalThis.browser` or `this.browser`?